lib/schema.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+module.exports = {
+  projects: {
+    url: '/accounts/{accountId}/projects',
+    requiredFields: ['accountId'],
+    list: {
+      type: 'GET'
+    },
+    create: {
+      type: 'POST',
+      requiredFields: ['name', 'sitePrimary']
+    },
+    read: {
+      url: '/{projectId}',
```

Comment:
Unless this is useful for mapping `projectId` specifically I guess, for easily mapping that name from the data as the "id" value. It's your call, but if the ids can be mapped generically that would save boilerplate schema code.